### ISSUE TITLE
Add high contrast mode toggle

### ISSUE THREAD

**Issue by katemoser** [katemoser/issue-helper#8]
Since this app is all about accessibility, we should have a toggle for viewing the application in high contrast

check out the DaisyUI docs for this: https://daisyui.com/components/theme-controller/
